### DISCUSSION TITLE
tmp tables not converted to raw tables

### DISCUSSION THREAD

**Discussion by anil-inito** [airbytehq/airbyte#32584] [Connector Questions]
I am trying to sync data from facebook marketing API but stopped manually the sync after 15 hrs. I was able to get data in temp table like this: ._airbyte_tmp_bei_ads_insights.
But the issue it is not converted to raw table "_airbyte_raw_ads_insights". However when I ran again the sync to sync remaining data, it got converted to raw table with remaingn data. So now I have 2 tables, temp table with major chunk of data and raw table with remaining data. 
Why this happened? Ideally the first sync should have been converted to original raw table and the later sync should be appended to it.

Can I manually merge them?

**Comment by marcosmarxm** [comment]
Did you try to reset first to clean up the tmp table?

**Reply by marcosmarxm** [reply]
Also what destination and version of connectors are you using?

[a comment by an automated account is not shown]
